parse.y

```diff
@@ -1956,6 +2037,28 @@ rb_parser_string_pointer(rb_parser_string_t *str)
     return str->ptr;
 }
 
+#ifndef UNIVERSAL_PARSER
+# define PARSER_STRING_GETMEM(str, ptrvar, lenvar, encvar) \
+    ((ptrvar) = str->ptr,                            \
+     (lenvar) = str->len,                            \
+     (encvar) = str->enc)                            \
```

Comment:
A macro ending with an empty line feels like a pitfall.
```suggestion
     (encvar) = str->enc)                            \
     /* end */
```

Reply:
I will remove `\` from the last line.